src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefaultCheckNullLabel.java

```diff
@@ -0,0 +1,53 @@
+/*
+MissingSwitchDefault
+
+
+*/
+
+//non-compiled with javac: Compilable with Java21
+package com.puppycrawl.tools.checkstyle.checks.coding.missingswitchdefault;
+
+public class InputMissingSwitchDefaultCheckNullLabel{
+
+    public void nestedSwitch1() {
+        int i = 1;
+        switch (i) {
+            case 1:
+                i++;
+                break;
+            case 2:
+                i--;
+                break;
+            default:
+                switch (i++) {
+                    case 2:
+                        i++;
+                        break;
+                    case 3:
+                        i--;
+                        break;
+                    case null:
+                }
+        }
+    }
+
+    public void nestedSwitch2() {
+        int i = 1, j = 2;
+        switch (i) {
+            case 1:
+                switch (j) {
+                    case 2:
+                        break;
+                    case 3:
+                        break;
+                    case null:
```

Comment:
Primitives cannot be null. You will need to take time to understand Java and switch statement/expression syntax, and make sure that these test inputs make sense and compile locally.